Tests/SwiftParserTest/translated/InvalidTests.swift

```diff
@@ -359,15 +359,10 @@ final class InvalidTests: ParserTestCase {
   }
 
   func testInvalid16a() {
-    // https://github.com/apple/swift/issues/43591
-    // Two inout crash compiler
     assertParse(
       """
       func f1_43591(a : inout 1️⃣inout Int) {}
```

Comment:
Leaving diagnosing multiple for ASTGen I assume?

Reply:
Yes